lib/commands/actions.js

```diff
@@ -43,6 +43,15 @@ commands.setOrientation = async function (orientation) {
   return await this.bootstrap.sendAction("orientation", params);
 };
 
+commands.setEmuOrientation = async function (adb, orientation) {
```

Comment:
This should be on the `extensions` or `helpers` object, as it is not technically a command.

Also, since it eventually gets called in this context of the driver, there is no need to pass in `adb`. `this.adb` will be the same here as in the calling code.